lib/basedriver/automations.js

```diff
@@ -0,0 +1,12 @@
+// Lists all the names of the automations that Appium supports
+const automations = {
```

Comment:
would prefer `automationName` throughout instead of `automation`

Reply:
how about Windows and Mac OS drivers?

Reply:
Windows and Mac drivers, along with android and ios, are handled by `platformName` not `automationName` (see https://github.com/appium/appium/blob/master/lib/appium.js#L62-L127).